eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,10 +1,25 @@
 stages:
   - stage: Build
+    dependsOn: []
+    # Purpose must be set at the level of the pipeline to enable this scenario
+    condition: eq(variables['PipelineType'], 'unified')
     jobs:
       - template: ../jobs/archetype-sdk-client.yml
 
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
```

Comment:
Add pipeline names ends with "- tests"

Reply:
If we do this then the only time this pipeline runs is in the context of tests... There will be no point in skipping it (below)

Reply:
Correct we don't want the " - tests" condition here (too many iterations in my head).